### ISSUE TITLE
setObjectForKey: object cannot be nil (key: acct)

### ISSUE THREAD

**Issue by dpfaffenbauer** [QuickBlox/quickblox-ios-sdk#491]
Hey,

something within [QBSession currentSession] throws this exception

```
 *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** setObjectForKey: object cannot be nil (key: acct)'
``` 

i don't know why. This only happens on iOS 8.4 (Simulator), on iOS 9.3 it works as expected. This happens within QMChatService:initWithServiceManager. I dont know what more i can tell you.. Is there any key value named "acct" inside the Quickblox Framework?

**Environment details**
iOS 8.4
Xcode 7.3.1

**Logs**
https://gist.github.com/dpfaffenbauer/152a6a49eab3913df9f8aa986ba08c24

**Steps to reproduce the behavior**
I don't really know... This only happens in my App. The Swift Demo works as expected. 

**Installed Pods**
```
    pod 'SVProgressHUD', '~> 2.0.3'
    pod 'TWMessageBarManager', '~> 1.7.2'
    pod 'SWRevealViewController', '~> 2.3.0'
    pod 'SSKeychain'
    pod 'Reachability'
    pod 'EZAudio'

    pod 'QuickBlox'
    pod 'QMCVDevelopment', :git => 'http://github.com/QuickBlox/QMChatViewController-ios', :commit => '258d18c542358f720cc437424d394fc012be318a'
    pod 'QMServicesDevelopment', :git => 'http://github.com/QuickBlox/q-municate-services-ios', :commit => '15fcc051cc280a9081bffe3c91ab3c82ca8ecfdc'
```

**Installed Carthage**
````
github "Alamofire/Alamofire" ~> 3.4
````
 
**Other Comments**
I did find a similar issue where this error is thrown (https://github.com/QuickBlox/quickblox-ios-sdk/issues/420). he said something about ARC, but ARC is turned on. So this is not the Problem.

Maybe you could give me some hints to find the solution for this problem.

**Comment by dpfaffenbauer** [comment]
[travelbuddy.zip](https://github.com/QuickBlox/quickblox-ios-sdk/files/286850/travelbuddy.zip)

Attached you will find a demo without any custom code (only the init) plus all deps (pods and carthage, you will need to call pod install first). Maybe you will the issue with that more easily.

**Comment by Raikerian** [comment]
Hi,

Please add exceptions breakpoint to see the exact line of code where issue occurres.

**Comment by dpfaffenbauer** [comment]
It fails in file QMChatService.m on line 54 (added a screenshot, maybe we use different versions of QMServicesDevelopment)

![bildschirmfoto 2016-06-02 um 08 38 07](https://cloud.githubusercontent.com/assets/5981845/15736031/59b6a060-289d-11e6-8a95-784b23c6e963.png)

**Comment by Raikerian** [comment]
Thank you for reporting the issue, I was able to reproduce it in the environment you've described and successfully fixed it. Fix will be available in the next release, approx. next week.

**Comment by dpfaffenbauer** [comment]
yeah great :), thank you

**Comment by Raikerian** [comment]
Fixed in 2.7.3.

**Comment by dpfaffenbauer** [comment]
Just tested it and it works, thank you very much :)
